frontend/src/Editor/QueryManager/QueryEditors/TooljetDatabase/DropDownSelect.jsx

```diff
@@ -127,6 +150,25 @@ const DropDownSelect = ({
             addBtnLabel={addBtnLabel}
             emptyError={emptyError}
             highlightSelected={highlightSelected}
+            foreignKeyAccess={foreignKeyAccess}
+            showRedirection={showRedirection}
+            columnInfoForTable={columnInfoForTable}
+            showColumnInfo={showColumnInfo}
+            showDescription={showDescription}
+            foreignKeyAccessInRowForm={foreignKeyAccessInRowForm}
+            isCellEdit={isCellEdit}
+            scrollEventForColumnValus={scrollEventForColumnValus}
+            organizationId={organizationId}
+            foreignKeys={foreignKeys}
+            setReferencedColumnDetails={setReferencedColumnDetails}
+            shouldShowForeignKeyIcon={shouldShowForeignKeyIcon}
+            cellColumnName={cellColumnName}
+            isInitialForeignKeyDataLoaded={isInitialForeignKeyDataLoaded}
+            setIsInitialForeignKeyDataLoaded={setIsInitialForeignKeyDataLoaded}
+            totalRecords={totalRecords}
+            setTotalRecords={setTotalRecords}
+            pageNumber={pageNumber}
+            setPageNumber={setPageNumber}
```

Comment:
I'm concerned about the number of props piling up, can we group them based on context to have better readability?

```js
pagination={{
  pageNumber,
  setPageNumber,
  totalRecords,
  setTotalRecords
}}

foreignKeyProps={{
  foreignKeyAccess,
  foreignKeyAccessInRowForm,
  foreignKeys,
  setReferencedColumnDetails,
  shouldShowForeignKeyIcon,
  isInitialForeignKeyDataLoaded,
  setIsInitialForeignKeyDataLoaded
}}

```